tests/Feature/Back/ProfileTest.php

```diff
@@ -30,7 +30,7 @@ public function can_see_profile()
         $response = $this->get('profile');
 
         $response->assertStatus(Response::HTTP_OK);
-        $response->assertSee($this->user->name);
+        $response->assertSee(e($this->user->name));
```

Comment:
When faker generates names like `Carl O'Neil`, `assertSee` fails.

Reply:
Hehe, i have this problem in other projects, didn't know about that `e()`
What does it exactly do?

Reply:
It runs the string into `htmlspecialchars` with `UTF-8` encode.